domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -461,7 +492,10 @@ private void init() throws Exception {
       startEmbeddedPostgres();
     }
 
-    client = io.vertx.ext.asyncsql.PostgreSQLClient.createNonShared(vertx, postgreSQLClientConfig);
+    PgConnectOptions connectOptions = createPgConnectOptions(postgreSQLClientConfig);
+
+    PoolOptions poolOptions = new PoolOptions().setMaxSize(5);
```

Comment:
Do we want to hard code the max size for the connection pool?

Reply:
We will probably have an env setting for this.